test/common/upstream/hds_test.cc

```diff
@@ -18,7 +18,10 @@
 #include "test/mocks/protobuf/mocks.h"
 #include "test/mocks/server/admin.h"
 #include "test/mocks/server/instance.h"
-#include "test/mocks/upstream/mocks.h"
+#include "test/mocks/upstream/cluster.h"
```

Comment:
not included in the build target, if it's used directly please add?

Reply:
oops, I forget to remove it. It's unused in this target